django_test_migrations/migrator.py

```diff
@@ -64,8 +64,10 @@ def __init__(
 
     def before(self, migrate_from: _Migration) -> ProjectState:
```

Comment:
We also need to adjust a type here. I suggest to create a named `Union` like so:

```python
_MigrationSpec = Union[_Migration, List[_Migration]]
```

Reply:
Thanks for your great comments.
Updating...